lib/widgets/movie_poster.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class MoviePoster extends StatelessWidget {
+  const MoviePoster({
+    super.key,
+    required this.pathToPosterImg,
+  });
+
+  final String pathToPosterImg;
+  static const double width = 200;
+  static const double height = 300;
+  static const double borderWidth = 2;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: width,
+      height: height,
+      child: Container(
+        decoration: BoxDecoration(
+          border: Border.all(width: borderWidth),
+          image: DecorationImage(
+            image: AssetImage(pathToPosterImg),
+            fit: BoxFit.fill,
+          ),
+        ),
+      ),
+    );
+  }
+}
```

Comment:
format the code to add the extra line that github is marking here, there is a config on settings > editor > general > ensure end of file line, that solves this issue

Reply:
sorted